.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 node_modules
 bower_components
+yarn.lock
```

Comment:
This should probably use unix line ending (add a newline)

Reply:
Thank you so much Mlunoe: now cropit works with jquery 3

Reply:
@klucznik did all the hard work though